src/views/portfolio/projects/ProjectList.vue

```diff
@@ -168,13 +168,35 @@
           dataOff: '\u2715'
         },
         columns: [
-          {
+        {
             title: this.$t('message.project_name'),
             field: "name",
             sortable: true,
             formatter(value, row, index) {
-              let url = xssFilters.uriInUnQuotedAttr("../projects/" + row.uuid);
-              return `<a href="${url}">${xssFilters.inHTMLData(value)}</a>`;
+              let name_url = xssFilters.uriInUnQuotedAttr("../projects/" + row.uuid);
+              let projectName = `<a href="${name_url}">${xssFilters.inHTMLData(value)}</a>`;
+              return projectName
```

Comment:
I'll take the liberty of making a few comments on this, even though I'm only a guest here.

The change is completely unnecessary. Also, the extra variable assignment before the return is unnecessary.

Reply:
agreed (this was just a leftover from a previous iteration where the tags where included in the project name column)